validation.go

```diff
@@ -131,7 +167,14 @@ type ValidationRegex struct {
 }
 
 func (v ValidationRegex) validate(value string) error {
-	// TBD
+	r, err := regexp.Compile(v.Pattern)
```

Comment:
you mean `*regexp.Regexp`instead of `string` right? that would mean the developer would have an extra step of creating the regex. but I agree this would be helpful with catching the invalid pattern early/